phpstan.neon

```diff
@@ -13,11 +13,8 @@ parameters:
         - lib/Test.php
         - lib/WorseReflection/Core/SourceCodeLocator/InternalStubs/*
         - lib/Extension/LanguageServerCodeTransform/Tests/Stub/*
-    reportUnmatchedIgnoredErrors: false
+    reportUnmatchedIgnoredErrors: true
     inferPrivatePropertyTypeFromConstructor: true
-    ignoreErrors:
-        - '#Yield can be used only with these return types: Generator, Iterator, Traversable, iterable.#'
```

Comment:
why removing this?

Reply:
Because we fixed all the errors.

Reply:
hm if that's the case then :+1: although I thought this was a specific issue with Amphp, but if it passes it passes :taco: